_extensions/r3-theme/includes/follow-rostools.qmd

```diff
@@ -0,0 +1,9 @@
+::: {.callout-tip appearance="minimal"}
+:star: Do you find this workshop material useful?
+
+Please consider following the [rostools GitHub
+organisation](https://github.com/rostools) to show your appreciation!
```

Comment:
Following is less useful and it will mean they see all the stuff happening in the repo on their landing page. Is there a particular reason you considered following vs starring?